targets/targets.json

```diff
@@ -571,6 +571,20 @@
         "release_versions": ["2", "5"],
         "device_name": "MKW41Z512xxx4"
     },
+    "RO359B": {
```

Comment:
@0xc0170 TARGET_RO359B folder is in the /targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/TARGET_MCU_K24F/ folder.
You meant about the definition in target.json? I used FRDM-K64F style on target.json.